test/functional/sync_plan/info_test.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/sync_plan'
+require_relative 'sync_plan_helpers'
+require_relative '../organization/organization_helpers'
+module HammerCLIKatello
+  describe SyncPlan::InfoCommand do
+    include OrganizationHelpers
+    include SyncPlanHelpers
+
+    it 'by ID' do
+      id = 1
+      api_expects(:sync_plans, :show) do |p|
+        p['id'] == id
+      end
+      arguments = "sync-plan info --id #{id}"
+      run_cmd(arguments.split)
+    end
+
+    it 'by organization id and sync plan name' do
+      name = "syncplan1"
+      org_id = 1
+      id = 1
+      # Make sure it calls to search for the correct sync plan among the plans in that org
+      # and gets that sync plan's id.
+      expect_sync_plan_search(org_id, name, id)
+
+      api_expects(:sync_plans, :show) do |p|
+        p['id'] == id
+      end
+      arguments = "sync-plan info --organization-id #{org_id} --name #{name}"
+      run_cmd(arguments.split)
+    end
+
+    it 'by organization name and sync plan name' do
```

Comment:
Organization options (name, label) should be a separate `describe` block to differentiate what you're testing.

```rb
describe "accepts organization" do
  it "name" do
    ...
  end
end
```

This then reads "SyncPlan::InfoCommand accepts organization name"